src/box/index.h

```diff
@@ -459,23 +459,23 @@ key_validate(const struct index_def *index_def, enum iterator_type type,
 	     const char *key, uint32_t part_count);
 
 /**
- * Check that the supplied key is valid for a search in a unique
- * index (i.e. the key must be fully specified).
+ * Check that the index_def->key_def is valid for a search in
```

Comment:
That's wrong again.
The supplied key - is `key` argument.
I think you should write something like:
```
 * Check that the supplied key is valid for a search in a unique
 * index, i.e. the key must be fully specified and correspond to
 * index_def->key_def.
```

Reply:
Done.